charts/cortex/templates/ingester/ingester-svc-headless.yaml

```diff
@@ -1,19 +0,0 @@
-apiVersion: v1
-kind: Service
-metadata:
-  name: {{ template "cortex-ingester.headlessname" . }}
```

Comment:
This is actually used in line 19 in `ingester-statefulset.yaml` - probably should not be deleted al together!